build/Dockerfile

```diff
@@ -1,45 +1,30 @@
-FROM ubuntu:15.04
-LABEL version=0.0.3
+FROM centos:6
+LABEL version=0.1
```

Comment:
Can you modify build/docker-compose.yaml to have the version match this version?  The CI should then pick up the new docker image, and you'd get testing of the old build system then.

Reply:
changin this label was a mistake - I actually didn't even see that there's a docker-compse file.